docs/fedora-installation-guide.md

```diff
@@ -1,6 +1,6 @@
-# Installing Clear Containers 3.0 Alpha on Fedora
+# Installing Clear Containers 3.0 on Fedora
 
-Clear Containers **3.0-alpha** is available for Fedora\* versions **24** and **25**.
+Clear Containers **3.0** is available for Fedora\* versions **24** and **25**.
```

Comment:
@gorozco1  I would prefer to keep this generic based on the major version, rather than referring to alpha or beta versions.